### ISSUE TITLE
Make logo clickable + add ALT text

### ISSUE THREAD

**Issue by jangondol** [pyconsk/pycon.sk#11]
**Current behavior**: at pycon.sk, the text "PyCon SK" in the upper left corner in the navigation bar is clickable but the logo right next to it isn't.

**Desired behavior**: the logo should be clickable and point to the same page as the text "PyCon SK". Also, ALT text should be provided.

**Comment by mbag** [comment]
What should ALT text state? **"Home"** or **"Index"** or something else?

**Comment by jangondol** [comment]
"PyCon SK Home"?

**Comment by mbag** [comment]
That's fine by me :) I'll get the patch out in a few minutes.

**Comment by mbag** [comment]
Please, review the code and if it's OK, close the issue and get it out to the production server. Thanks.
